libs/pictureCard/pictureCard.css

```diff
@@ -0,0 +1,115 @@
+.card {
+  border: 1px solid var(--udexColorGrey2);
+  box-shadow: 0 2px 4px rgb(34 54 73 / 20%);
+  background-color: var(--background-color);
+  height: var(--article-tile-height);
+  border-radius: var(--sapTile_BorderCornerRadius);
+  overflow: hidden;
+  position: relative;
+
+  & img {
+    width: 100%;
+    height: 130px;
+    aspect-ratio: 4 / 3;
+    object-fit: cover;
+  }
+
+  & .cardcontent {
+    display: block;
+    padding: var(--udexSpacer16);
+
+    & .template {
+      font-style: normal;
+      font-family: var(--udexTypographyBodyXSFontFamily);
```

Comment:
I find this CSS to render the card quite bloated. I'll trim it once we're OK with the refactor logic.